### PR TITLE
Add initial content

### PR DESCRIPTION
**Pull request by lucywyman**
This is based off of Ethan's original PR, stripping out all the PDK
template we don't need and adding a CODEOWNERS file.

Supersedes #1

### CODE REVIEW

File: files/task_helper.ps1 [hunk 1]
```diff
@@ -0,0 +1,261 @@
+# pull some more reusable bits from here
+# https://github.com/puppetlabs/puppetlabs-bootstrap/blob/master/tasks/windows.ps1
+Add-Type -AssemblyName System.ServiceModel.Web, System.Runtime.Serialization
+
+function ConvertTo-JsonString($string)
```
Comment (jpogran) [inline files/task_helper.ps1:5]: this is the only function to keep from this file. the rest were shipped as part of bolt until 3.0, where they were removed when support for powershell 2 was removed

**Comment by jpogran** [comment]
@lucywyman and I have some ideas from feedback to add other things to this helper. working on that

**Comment by lucywyman** [comment]
@jpogran Awesome! I feel like putting up your own PR and closing this one is probably easiest, but feel free to push to my branch if you'd prefer.

**Comment by jpogran** [comment]
Superseded by https://github.com/puppetlabs/puppetlabs-powershell_task_helper/pull/3

**Pull request closed without merging**